OnlyM/Services/ImagesCache/ImageCache.cs

```diff
@@ -15,49 +15,57 @@ internal class ImageCache
         private const int MaxImageWidth = 1920;
         private const int MaxImageHeight = 1080;
 
-        private readonly ConcurrentDictionary<string, ImageAndLastUsed> _cache = 
+        private readonly ConcurrentDictionary<string, ImageAndLastUsed> _cache =
             new ConcurrentDictionary<string, ImageAndLastUsed>(StringComparer.OrdinalIgnoreCase);
 
         public BitmapSource GetImage(string fullPath)
         {
-            if (!_cache.TryGetValue(fullPath, out var result))
+            if (!File.Exists(fullPath))
```

Comment:
Could we use `new FileInfo(fullPath)` here for performance? We could remove `File.Exists` and `File.GetLastWriteTimeUtc`

Reply:
Indeed, that makes sense. I will fix this.